docs/how-to-guides/image-processing/bundle-adjustment-in-isis.md

```diff
@@ -11,16 +11,32 @@ Bundle adjustment inherently requires multiple observations, and ISIS expects th
 ls -d -1 "$PWD/"*.cub > image_list.lis
```

Comment:
Am I missing the part where the person downloads the images and control network to run through this tutorial?